common/FileUtil.cpp

```diff
@@ -387,6 +388,7 @@ namespace FileUtil
     std::unique_ptr<std::vector<char>> readFile(const std::string& path, int maxSize)
     {
         auto data = std::make_unique<std::vector<char>>(maxSize);
+        data->resize(0);
```

Comment:
So, before this was actually appending the contents after mazSize zeros? So, e.g. the case in wsd/wopi/WopiProxy.cpp StorageBase::StorageType::FileSystem was broken?

Reply:
Yes, it seems so. But since the response was an empty file (all null), it had no ill-effect.